fix for new imagemagick include dir structur
Joerg Bornkessel hd_brummy@gentoo.org 2008 Aug 04
diff -Naur skinenigmang-0.0.6.orig/Makefile skinenigmang-0.0.6/Makefile
--- skinenigmang-0.0.6.orig/Makefile	2008-08-04 22:39:06.000000000 +0200
+++ skinenigmang-0.0.6/Makefile	2008-08-04 22:39:51.000000000 +0200
@@ -139,6 +139,7 @@
 ifdef HAVE_IMAGEMAGICK
 OBJS += bitmap.o
 LIBS += -lMagick++
+INCLUDES += -I/usr/include/ImageMagick
 endif
 
 ifdef HAVE_FREETYPE
